File: src/SOP8/Utility/dnsData/CommonCode/CodeTypeData.cs
using System.Collections.Generic;

namespace dnsData.CommonCode
{
    public abstract class CodeTypeData
    {
        protected List<int> m_collections = new List<int>();

        public IEnumerable<int> Collection
        {
            get { return m_collections; }
        }

        public abstract CommonCode.CodeType CodeType
        {
            get;
        }
    }
}

File: src/SOP8/광양/EgovLogin/base-utility/src/main/java/egovframework/base/data/common/code/CodeTypeData.java
package egovframework.base.data.common.code;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;

import egovframework.base.data.common.code.CodeType;

public abstract class CodeTypeData {
	protected List<Integer> collections = new ArrayList();

	public List<Integer> getCollection() {
        return collections;
    }

    public abstract CodeType getCodeType();
}
